packages/widget/src/components/address-input/address-input.ts

```diff
@@ -0,0 +1,32 @@
+import { LitElement, html } from 'lit';
+import type { HTMLTemplateResult } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { styles } from './styles';
+
+@customElement('address-input')
```

Comment:
considering all elements live in shared namespace, I would suggest prefixing with "sygma-"

Reply:
got it